src/components/About.astro

```diff
@@ -1,153 +1,53 @@
 <section id="about" class="scroll-mt-20">
   <div class="mx-auto max-w-7xl pt-12 px-4 sm:px-6 lg:px-8">
-    <div class="lg:text-center">
+    <div class="lg:text-left">
       <p
         class="mt-2 text-3xl font-bold leading-8 tracking-tight text-gray-900 sm:text-4xl md:text-5xl"
       >
         What to expect at HackHPI 2023
       </p>
-      <p class="mt-4 max-w-2xl text-xl text-gray-500 lg:mx-auto">
-        Lorem ipsum dolor sit amet consect adipisicing elit. Possimus magnam
-        voluptatum cupiditate veritatis in accusamus quisquam.
+      <p class="mt-4 text-xl text-gray-500 lg:mx-auto">
+        A functioning government is crucial for ensuring efficient public
+        services. It is necessary that government agencies adapt their
+        infrastructure and used technologies to the increasingly digitized
+        society to effectively serve their citizens.
       </p>
-    </div>
-
-    <div class="mt-10">
-      <dl
-        class="space-y-10 md:grid md:grid-cols-2 md:gap-x-8 md:gap-y-10 md:space-y-0"
+      <p class="mt-4 text-xl text-gray-500 lg:mx-auto">
+        In today's world, where sensitive information and critical
+        infrastructure are increasingly connected to the internet, it is
+        essential that government agencies and institutions are protected
+        against cyber threats.
+      </p>
+      <p class="mt-4 text-xl text-gray-500 lg:mx-auto">
+        However, as many people experience every day, in the field of
+        digitalization, government and public institutions are still lagging
+        behind. This year is about developing secure solutions that can help to
+        improve and democratize these areas.
+      </p>
+      <p class="mt-4 text-xl text-gray-500 lg:mx-auto">
+        For a weekend of hacking, we will bring motivated students from
+        different backrounds together at HPI, just outside of Berlin. By
+        providing world class sponsors including workshops and access to their
+        data and APIs, a great location and plenty of caffeine to fuel your
+        brain we will create a space for creativity, inspiring ideas and a
+        passion for trying out new things.
+      </p>
+      <br />
+      <p
+        class="mt-2 text-3xl font-bold leading-8 tracking-tight text-gray-900 sm:text-4xl md:text-xl"
       >
```

Comment:
You can replace the `<br />` with a greater `margin-top`, and we probably want to adapt the text size on different screen sizes (in your current change, it is just as big as the title text on smaller screens).

```suggestion
      <p class="mt-8 text-xl font-bold tracking-tight text-gray-900">
```